flink-connectors/flink-connector-kafka-base/src/test/java/org/apache/flink/streaming/connectors/kafka/FlinkKafkaConsumerBaseTest.java

```diff
@@ -696,6 +696,7 @@ public void clear() {
 
 		@Override
```

Comment:
Change also the `addAll()`.

Reply:
And also,  there is the `update()` method in the `ListState`  interface that should be changed in all implementations.

Reply:
fixing